<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdWidget</class>
 <widget class="QWidget" name="ThresholdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThresholdWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Threshold</string>
   </property>
   <widget class="QSlider" name="sliderThreshold">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>150</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditThreshold">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>150</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAutoThreshold">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Threshold</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAutoThresLevel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelThresholdNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAutoThresholdResult">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxReverseThres">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse Threshold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
